services/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -104,7 +104,7 @@ public class AsyncQueryForwardingServlet extends AsyncProxyServlet implements Qu
   private static final String PROPERTY_SQL_ENABLE = "druid.router.sql.enable";
   private static final String PROPERTY_SQL_ENABLE_DEFAULT = "false";
 
-  private static final int CANCELLATION_TIMEOUT_MILLIS = 500;
+  private static final int CANCELLATION_TIMEOUT_MILLIS = 1000;
```

Comment:
Should we bump it up to 5 seconds ?
The reason is for a cluster that is fully loaded, a cancel call would help ease the pressure on the broker. 
cc @abhishekagarwal87?

Reply:
How did you arrive at the 5 second number?

Reply:
Guesstimate mostly.

Reply:
Updated to 5 seconds.